src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/ReadsPipelineSpark.java

```diff
@@ -142,6 +141,9 @@
         return BaseRecalibrationEngine.BQSR_REFERENCE_WINDOW_FUNCTION;
     }
 
+    @Override
+    public boolean useVariantAnnotations() { return true;}
```

Comment:
Need to also override `getDefaultVariantAnnotationGroups()` and `makeVariantAnnotations()` in this class, as you did for `HaplotypeCallerSpark`